pkg/config/apm.go

```diff
@@ -151,6 +152,22 @@ func setupAPM(config Config) {
 		return out
 	})
 
+	config.SetEnvKeyTransformer("apm_config.custom_tags", func(in string) interface{} {
+		var out []map[string]string
+		if err := json.Unmarshal([]byte(in), &out); err != nil {
+			log.Warnf(`"apm_config.custom_tags" can not be parsed: %v`, err)
+		}
+		return out
+	})
+
+	config.SetEnvKeyTransformer("apm_config.custom_span_names", func(in string) interface{} {
```

Comment:
Note: the `out` var should likely be `[]map[string]interface{}` instead (if you follow my suggestion above).

Reply:
I actually overlooked this file by accident and forgot to delete the custom_span_names after I made the changes to make the `custom_tags` a map instead of a list. Right now in the staging_conf.yaml I have the `custom_tags` field setup like this:

```
 custom_tags:
    "sqlquery": ["georegion", "costcenter"]
    "redis": ["country"]
```

With this, can I just do `var out []map[string][]string` under this function: `config.SetEnvKeyTransformer("apm_config.custom_tags", func(in string) interface{} {` ?